pom.xml

```diff
@@ -47,6 +47,39 @@
 				<type>test-jar</type>
 				<scope>test</scope>
 			</dependency>
+			<dependency>
```

Comment:
Can't have this dependencies in the parent pom, they belong to the omod child pom